test/logger-usage/build.gradle

```diff
@@ -0,0 +1,27 @@
+import org.elasticsearch.gradle.precommit.PrecommitTasks
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
```

Comment:
Can this use the `elasticsearch.build` plugin? I noticed the code is using eg File instead of Path, which would have been caught by forbidden apis. I realize it is sort of a chicken-and-egg problem, but I think we should be able to compose this in such a way that the task which depends on this project is not added? Perhaps it is worth a hardcoded conditional on the project path in PrecommitTasks.